src/components/logs/EditLogModal.js

```diff
@@ -1,16 +1,37 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
```

Comment:
you're not destructuring PropTypes, like you're destructuring { connect } an { useState, useEffect } ////